pkg/skaffold/version/version.go

```diff
@@ -23,33 +23,38 @@ import (
 
 	"github.com/blang/semver"
 	"github.com/pkg/errors"
+
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/latest"
 )
 
 var version, gitCommit, gitTreeState, buildDate string
 var platform = fmt.Sprintf("%s/%s", runtime.GOOS, runtime.GOARCH)
+var configVersion = latest.Version
```

Comment:
why not just use latest.Version directly?

Reply:
ah yeah I was trying something else here originally, you're right I should